app/demoservice/templates/base.html

```diff
@@ -27,13 +28,27 @@
     </nav>
   </div>
 </header>
-{% if messages %}
-  <ul class="messages">
-    {% for message in messages %}
-      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
-    {% endfor %}
-  </ul>
-{% endif %}
+  {% if messages %}
+    <div class="p-strip">
```

Comment:
When QA'ing this strip causes a lot of space. Could you please add ` is-shallow`.